documentation/experimental_features.md

```diff
@@ -8,6 +8,113 @@ API might change, requiring the user to update their code or configuration. The
 Bolt team attempts to make these changes painless by providing useful warnings
 around breaking behavior where possible. 
 
+## Configuration policies and the policy command
+
+This feature was introduced in [Bolt 3.21.0](). 
+
+### What are policies?
+
+Configuration policies, or policies for short, are public classes that can be
+applied directly to one or more nodes. Policies are Puppet code stored in the
+`manifests/` directory of modules on the modulepath. They can be applied
+directly to targets just like other Puppet code.
+
+### Policies in Bolt project yaml
+
+A Bolt project's configuration file `bolt-project.yaml` has a `policies` key
+which lists the policies available to a project. A Puppet class becomes a policy
+when it's listed under the `policies` key, and if the `policies` key is empty
+then no policies will be available for Bolt to apply.
+
+Policies may be manually added to the `policies` key in `bolt-project.yaml` or
+managed via Bolt commands.
+
+Example bolt-project.yaml
+```
+---
+name: boltproject
+modules: []
+policies:
+- boltproject::admin
+- boltproject::sshkeys
+```
+
+### Policy management overview
+
+Policies can be managed through the `policy` command, with `apply`, `new` and
+`show` subcommands or the corresponding Powershell cmdlets `Invoke-BoltPolicy`,
+`Get-BoltPolicy`, and `New-BoltPolicy`. Use '-h' with any of the policy
+subcommands for more detail or `Get-Help` followed by any Powershell cmdlet
+name.
+
+#### Creating new policies 
+> Policy names must follow [class naming
+> conventions](https://puppet.com/docs/puppet/7/lang_reserved.html#classes-and-defined-resource-type-names).
+
+> Policy names must also be in the form `<PROJECT NAME>::<POLICY NAME>`. This is
+> referred to as `<POLICY NAME>` below.
+
+You can create a new policy with `bolt policy new <POLICY NAME>`, which performs
+two actions for you:
+
+1. `<POLICY NAME>` is added to the `policies` key in `bolt-project.yaml`.
+2.  An empty class is created at `<PROJECT DIRECTORY>/manifests/<POLICY
+    NAME>.pp` that you can then populate. Under the hood, Bolt creates a single
+    line of Puppet code "include #{policies.join(,)}" that will be compiled and
+    applied to the provided targets, and will log that code at the debug level.
+
```

Comment:
```suggestion
For example, to create a new project-level policy named `bolt::project::user`, run:
```

Each of the examples should have a quick blurb about what it's doing.

Reply:
Covered the blurbs I think.